pkg/operator/ceohelpers/bootstrap_test.go

```diff
@@ -0,0 +1,362 @@
+package ceohelpers
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	corev1listers "k8s.io/client-go/listers/core/v1"
+	"k8s.io/client-go/tools/cache"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+
+	"github.com/openshift/cluster-etcd-operator/pkg/operator/operatorclient"
+)
+
+func Test_GetBootstrapScalingStrategy(t *testing.T) {
```

Comment:
(No action required) Maybe this testing would be simplified if the method under test was refactored?

```golang
func GetBootstrapScalingStrategy(...) (...) {
  namespace := ...
  ...
  spec := ...
  ...
  return scalingStrategy(spec, namespace)  
}
```

Not much value in unit testing the API calls, so maybe only scalingStrategy would need to be tested.

Reply:
I had a similar thought while writing it and also came to the conclusion that it was maybe too granular to bother with